tests/python/contrib/test_hexagon/conv2d/test_conv2d_conv2d.py

```diff
@@ -190,12 +189,13 @@ def test_conv2d(
         k_split_factor,
         h_split_factor,
         dtype,
-        target,
     ):
         """conv2d-conv2d test"""
         # TODO: no support for padding in conv2d #2
         pad2 = 0
 
+        target = "llvm"
```

Comment:
yeah I reverted it. I did since I don't like the noise in the skip tests but we should fix that in TVM generally instead of hard coding the target here.

Reply:
I go back and forth on that part.  I tend to prefer the explicitly reported and skipped tests, because otherwise a skipped test can appear as a successful test.  But I also agree that having lists upon lists of skipped tests isn't useful for an interactive environment, and just adds noise.